Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -47,11 +47,27 @@ class LayoutManager: NSLayoutManager {
             enumerateLineFragments(forGlyphRange:listGlyphRange) { (rect, usedRect, textContainer, glyphRange, stop) in
                 let lineRect = rect.offsetBy(dx: origin.x, dy: origin.y)
                 let lineRange = self.characterRange(forGlyphRange: glyphRange, actualGlyphRange: nil)
+                guard let textList = textStorage.textListAttribute(atIndex: lineRange.location)
+                    else {
+                        return
+                }
+                let number = textStorage.itemNumber(in: textList, at: lineRange.location)
                 let isStartOfLine = textStorage.isStartOfNewLine(atLocation: lineRange.location)
                 var attributes = textStorage.attributes(at: lineRange.location, effectiveRange: nil)
-                attributes[NSParagraphStyleAttributeName] = NSParagraphStyle.Aztec.defaultParagraphStyle
+                attributes[NSParagraphStyleAttributeName] = ParagraphStyle.default
+                attributes.removeValue(forKey: NSUnderlineStyleAttributeName)
+                attributes.removeValue(forKey: NSStrikethroughStyleAttributeName)
+                attributes.removeValue(forKey: NSLinkAttributeName)
+                if let font = attributes[NSFontAttributeName] as? UIFont {
+                    var traits = font.fontDescriptor.symbolicTraits
+                    traits.remove(.traitBold)
+                    traits.remove(.traitItalic)
+                    let descriptor = font.fontDescriptor.withSymbolicTraits(traits)
+                    let newFont = UIFont(descriptor: descriptor!, size: font.pointSize)
+                    attributes[NSFontAttributeName] = newFont
+                }
```

Comment:
Are we suuuure this is the best spot to alloc a new Font / update few values here and there? (considering this method's name is `drawBackground`)